src/Uno.UI/Microsoft/UI/Xaml/Controls/ProgressBar/ProgressBar.cs

```diff
@@ -234,16 +255,40 @@ private void UpdateWidthBasedTemplateSettings()
 			templateSettings.ContainerAnimationStartPosition = indeterminateProgressBarIndicatorWidth * -1.0; // Position at -100%
 			templateSettings.ContainerAnimationEndPosition = indeterminateProgressBarIndicatorWidth * 3.0; // Position at 300%
 
-			templateSettings.ContainerAnimationStartPosition2 = indeterminateProgressBarIndicatorWidth2 * -1.5; // Position at -150%
-			templateSettings.ContainerAnimationEndPosition2 = indeterminateProgressBarIndicatorWidth2 * 1.66; // Position at 166%
+			templateSettings.Container2AnimationStartPosition = indeterminateProgressBarIndicatorWidth2 * -1.5; // Position at -150%
+			templateSettings.Container2AnimationEndPosition = indeterminateProgressBarIndicatorWidth2 * 1.66; // Position at 166%
```

Comment:
The rename here is a breaking change introduced in https://github.com/microsoft/microsoft-ui-xaml/pull/2362